test/core/surface/lame_client_test.cc

```diff
@@ -123,12 +123,7 @@ TEST(LameClientTest, MainTest) {
                                 grpc_core::CqVerifier::tag(1), nullptr);
   ASSERT_EQ(GRPC_CALL_OK, error);
 
-  // Filter stack code considers this a failed to receive initial metadata
```

Comment:
I assume this was a v2 artifact, and that the v3 stack reacts the same way that the legacy stack does?